test/fixtures/jsonschema.sql

```diff
@@ -0,0 +1,260 @@
+-- from gavinwahl/postgres-json-schema commit 5a257e19a1569a77b82e9182b0b7d9fc8b6f6382
+
```

Comment:
Regarding credit, how about if we copy the contents of https://github.com/gavinwahl/postgres-json-schema/blob/master/LICENSE here.

Reply:
Sounds good. Done.